source/includes/steps-export-pipeline-to-language.yaml

```diff
@@ -42,20 +42,33 @@ content: |
      selected.
 ---
 title: Include import statements, if desired.
+optional: true
 level: 4
 ref: include-import-statements
 content: |
-  *(Optional)*: Check the :guilabel:`Include Import Statements` option
+  Check the :guilabel:`Include Import Statements` option
   to include the required import statements for the language selected.
 ---
-title: Click :guilabel:`Copy`.
+title: Include driver syntax, if desired.
+optional: true
+level: 4
+ref: include-driver-syntax
+content: |
+  Check the :guilabel:`Include Driver Syntax` option to include 
+  application code for the language selected. If you include driver 
+  syntax, the copyable code reflects :ref:`project 
+  <query-bar-project>`, :ref:`sort <query-bar-sort>`, :ref:`maxtimems 
+  <query-bar-max-time-ms>`, :ref:`collation <query-bar-collation>`, 
+  :ref:`skip <query-bar-skip>` and :ref:`limit <query-bar-limit>` 
+  options. 
+---
+title: Click :icon-fa5:`copy`.
 level: 4
 ref: click-copy
 content: |
-  Click the :guilabel:`Copy` button at the top-right of the pipeline
-  to copy the pipeline for the selected language to your clipboard.
-  You can now integrate and execute your created pipeline into your
-  application.
+  Click :icon-fa5:`copy` at the top-right of the pipeline to copy the 
```

Comment:
[ out-of-scope ] While this icon change seeks to be helpful, it is B&W and mirrored compared to the screenshot icon (which is green). I know this is likely a site-wide design choice (B&W for readability), but it did cause me to search for matching icon in screenshot for a good minute before realize color mismatch.

Reply:
see [PR](https://github.com/mongodb/docs-tools/pull/488) for the icon. Once the PR is merged, I'll fix the icon.

Reply:
Nice, looks good!